src/scss/ucf-promo.scss

```diff
@@ -32,9 +33,9 @@ $screen-lg-min: 1200px;
 .promo-horizontal {
   padding: 30px;
 
-  h2 {
+  .promo-header {
     color: #fff;
-    font-family: 'Knockout 49 A','Knockout 49 B',Helvetica,Arial,sans-serif;
+    font-family: $font-family-condensed;
     font-size: 50px !important;
```

Comment:
I added a task in teamwork to clean up the !important references after the main site update.